### ISSUE TITLE
About page

### ISSUE THREAD

**Issue by OdinFK** [czarandy/mtgptresults#97]
I would like to create an about page, mostly with information about methology, but maybe some information about the project, too. The structure would be very easy, basically an FAQ, although I will not call it so as there just are no /frequently/ asked questions.

Shall I just create a .json file, or what is the preferred way to realize this?

**Comment by czarandy** [comment]
That info seems good to include in the README in this project.

**Comment by OdinFK** [comment]
Right. I think eventually we should put that on the website then. Most people will probably not look at some file in a github repository.

**Comment by czarandy** [comment]
It's linked right from the main page, and the main view on github shows you the README, so it's not exactly a random file.

**Comment by OdinFK** [comment]
I didn't mean this in any way as an attack. I just assume that many non-tech-affine people will presume the Github link goes to some coding stuff, that is basically Chinese for them.

**Comment by czarandy** [comment]
We can change the text that links to the github readme. I still think it makes more sense to include that info here, since this is the "behind the scenes" of how the site is generated.

**Comment by mtgmagicalhack** [comment]
@OdinFK @czarandy Is this still active? 

My 2 cents: 
* Technical stuff should be in the README.md at GitHub.
* "About us" information could be on website, but I would not know what content to put there.

**Comment by OdinFK** [comment]
I think the site should definitely have an 'About' section. It is cool to have this site, but without an about section the user cannot put this into context. What is the scope? Is this work in progress? Is this constantly updated? etc.
